src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -260,7 +260,12 @@ public string GetActiveFilterCriteria()
 
       public void GetFormatBasedOnCurrentSheet()
       {
-         _dataSourceFile.AvailableFormats = _importer.CalculateFormat(_dataSourceFile, _columnInfos, _metaDataCategories, _currentSheetName).ToList();
+         var availableFormats  = _importer.CalculateFormat(_dataSourceFile, _columnInfos, _metaDataCategories, _currentSheetName).ToList();
+         
+         if(availableFormats.Count <1 )
```

Comment:
if(!availableFormats.Any()) is typically what is used here. Also resharper should be used in this file

Reply:
done and format also fixed through Resharper